UnityProject/Assets/Scripts/Items/Others/Jetpack.cs

```diff
@@ -67,6 +67,10 @@ private void PushUpdate()
 			if (player.PlayerSync.IsPressedServer)//Is movement pressed?
 			{
 				PushPlayerInFacedDirection(player, gasContainer, gasReleaseOnUse);
+				if (releaseVisual != null)
+				{
+					Spawn.ServerPrefab(releaseVisual, gameObject.AssumedWorldPosServer());
```

Comment:
is this performant enough?

Reply:
It spawns a single splatter every 0.5 seconds if there is any movement on the server, and splatters handle their removal client side.

This shouldn't be a concern unless you're spawning hundreds of them, which would cause GC to pile up then.